src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -694,10 +694,7 @@ private void AddNativeLibraries (ArchiveFileList files, string [] supportedAbis)
 				return;
 			}
 
-			NdkTools? ndk = NdkTools.Create (AndroidNdkDirectory, Log);
-			if (ndk == null) {
-				return; // NdkTools.Create will log appropriate error
-			}
+			NdkTools ndk = NdkTools.Create (AndroidNdkDirectory, logErrors: false, log: Log);
```

Comment:
@jonathanpeppers: should we similarly check `Log.HasLoggedErrors` here as well?

Reply:
Ok, whoops I put this check back, so it should be closer to what we had before.